config/variables.env

```diff
@@ -0,0 +1,3 @@
+GOOGLE_CLIENT_ID = "318441039578-ghflj2cgc1g6aj02gqrg8pgd6mlf0kb2.apps.googleusercontent.com"
```

Comment:
Please never push secret keys and client ids onto any git 
use an example file for examples of those 
and share those keys personally,